react/components/Map.js

```diff
@@ -204,8 +204,10 @@ class Map extends Component {
       this.createNewMarkers()
     }
 
+    const bounds = this.map.getBounds()
```

Comment:
Is there any situation where `this.map` could be `undefined`/`null`?

Reply:
Nope, it will always be there.